app/models/group.rb

```diff
@@ -1,10 +1,10 @@
 class Group < ApplicationRecord
   belongs_to :author, class_name: 'User'
-  # has_and_belongs_to_many :entities
-  # has_one_attached :icon
+  has_and_belongs_to_many :entities
+  has_one_attached :icon
 
-  has_many :entities_group_to_joins, dependent: :destroy
-  has_many :entities, through: :entities_group_to_joins
+  # has_many :entities_group_to_joins, dependent: :destroy
+  # has_many :entities, through: :entities_group_to_joins
```

Comment:
Kindly remove this unused code for it makes your code base look untidy and may cause you to have errors.